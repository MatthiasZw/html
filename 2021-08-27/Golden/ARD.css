/* === Allgemeinse Styling === */
html, *{
    margin: 0;
    padding: 0;
    border:none;
    font-family: Arial, Helvetica, sans-serif;
    font-size: 16px;
    color: #333;
}

.wrapper{
    margin: 0 auto;
}

img{
    width: 100%;
}

body {
    background: #73a4e0 url(../images/base_xs.jpg)
    no-repeat top center fixed;
}

@media screen and (min-width:769px) {
    body{
        background: url(../images/base_xl.jpg);
    }
    .wrapper{
        width: 996px;
    }
}

/* === Navigation === */
#mainnav {
    width: 100%;
    background-color: rgba(66,88,126,.7);
    border-bottom:1px solid #d9fffc;
}

#mainnav div.wrapper{
    display: flex;
}

#mainnav ul{
    display: none;
    list-style: none;
}

#mainnav ul:first-child{
    margin-right: 50px;
}

#mainnav ul:last-of-type{
    font-weight: bold;
}

#mainnav a{
    padding: 11px 5px 8px;
    font-size: 0.9rem;
}

#mainnav a:link,
#mainnav a:visited,
#mainnav a:active{
    color: #fff;
    text-decoration: none;
}

#mainnav a:hover{
    color: #333;
    background-color: #fff;
}



@media screen and (min-width:769px){
    #mainnav ul{
        display: flex;
        padding:10px 2px 5px;
    }
        #navbutton {
            display: none;
   }
}

/* Logo in Navigation */

#branding{
    width: 70px;
    height: 35px;
    background: url(../images/ardlogo_weiss.png)
    no-repeat 0 0;
    margin-left: auto;
    margin-right: 20px;
}

#branding h2{
    text-indent: -999999px;
}

#branding a{
    display: inline-block;
    width: 70px;
    height: 35px;
}

/* Klickfl√§che bei ARD LOGO */
#branding a:hover{
    color: rgba(1,1,1,1) !important;
    background-color: transparent;
}

/*  === main_head === */

#main_head{
    margin-bottom: 30px;
}

#main_logo img{
width: 280px;
}

#main_head form{
    margin-top: 20px;
}

#main_head input{
    background:#fff url(../images/lens.svg) no-repeat center right;
    background-size: 30px 30px;
    height: 36px;
    width: 260px;
    padding: 3px 40px 3px 3px;
    font-size: .9rem;
}

@media screen and (min-width: 769px) {
    #main_head{
        display: flex;
        justify-content: space-between;
    }
}

/*  === main_content === */
.main_content{
    background: #fff;
    padding: 10px;
    margin-top: 10px;
}

.row{
    width: 100%;
}

h2{
    font-size: 2rem;
    margin-bottom: 20px;
}

header h2{
    font-family: "Times New Roman",serif;
}

h4{
    margin-top: 10px;
}
p{
margin: 15px 0;
}

p:first-child{
    margin-top: 0;
}

p.teaser{
    border-top: 1px solid #42587e;
    padding: 10px 0;
    margin: 0;
}

p.teaser:last-child{
    border-bottom: 1px solid #42587e;
}

@media screen and (min-width: 769px){
    .row{
        display: flex;
        justify-content: space-between;
    }
    .col-2{
        width: 478px;
    }
    .col-3{
        width:312px;
    }
    .col-4{
        width:229px;
    }
}

/* ===  Footer  === */
footer{
    background-color: #001a4b;
    margin-top: 20px;
    padding-top: 30px;
    padding-bottom: 12px;
}

footer h4,
footer p,
footer a{
    color: #d9fffc;
    text-decoration: none;
}

footer ul li{
    list-style: none;
    margin-top: 5px;
    }

footer .imprint{
    margin-top: 15px;
}

footer img{
    width: 200px;
    height: auto;
}

@media screen and (min-width: 769px){
    footer .imprint{
        display:flex;
    }
    footer ul.imprint li {
        margin-right: 10px;
    }
    footer .copy{
        display: flex;
        justify-content: space-between;
        margin-top: 10px;
    }
}