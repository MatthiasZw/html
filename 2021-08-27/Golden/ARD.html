<!DOCTYPE html>
<html lang="de">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>ARD Startseite</title>
    <link rel="stylesheet" href="css/styles.css">
</head>
<body>
    
<!-- Navigation -->

<nav id="mainnav">
    <div class="wrapper">
        <ul>
            <li><a href="#">ARD Home</a></li>
            <li><a href="#">Nachrichten</a></li>
            <li><a href="#">Sport</a></li>
            <li><a href="#">Börse</a></li>
            <li><a href="#">Ratgeber</a></li>
            <li><a href="#">Wissen</a></li>
            <li><a href="#">Kultur</a></li>
            <li><a href="#">Kinder</a></li>
            <li><a href="#">ARD Intern</a></li>
        </ul>

        <ul>
            <li><a href="#">Fernsehen</a></li>
            <li><a href="#">Radio</a></li>
            <li><a href="#">Mediathek</a></li>
        </ul>

        <div id="branding">
            <a href="#">
                <h2>ARD</h2>
            </a>
        </div>

        <div id="navbutton">
            <button>Nav</button>
        </div><!-- /.navbutton -->
    </div> <!-- /.wrapper -->
</nav><!-- /.mainnav -->


<main class="wrapper">

<section id="main_head">
    <div id="main_logo">
        <a href="#"><img src="images/logo_ard.png" alt="ARD Logo"></a>
    </div>

    <form action="">
        <input type="search" name="search" placeholder="Suche in der ARD Mediathek">
    </form>


</section>

<!-- 2-Spalter -->

<section class="main_content">
    <header>
        <h2>Nachrichten</h2>
    </header>
    
    <div class="row">
        <div class="col-2">
            <img src="images/organ.jpg" alt="Organspende - darum geht es">
        </div>
        <div class="col-2">
            <p>Abstimmung im Bundestag</p>
            <h2>Organspende - darum geht es</h2>
            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quod ullam iste hic blanditiis? Nam expedita tempora iusto incidunt ad aliquam veritatis impedit quisquam enim. Cum nisi quia expedita laudantium impedit!</p>
            <p class="teaser">Berlin: Vor der Abstimmung zur Organspende | audio</p>
            <p class="teaser">Diskussion um Organspende:"Das muss schon Spende bleiben", 12.01.2020</p>     
            <p class="teaser">Zahl der Organspenden ging 2019 leicht zurück, 13.01.2020</p>
        </div><!-- /.col-2 -->
    </div><!-- /.row -->
</section>

<!-- 3-Spalter -->
<section class="main_content">
<div class="row">
    <div class="col-3">
        <figure>
            <img src="images/handball1.jpg" alt="Handball EM">
            <figcaption>Handball EM</figcaption>
        </figure>
        <p>TSG 1860 Hoffenheim im teamcheck: Eine Hinrunde mit Licht und Schatten</p>
    </div>
    <div class="col-3">
        <figure>
            <img src="images/handball2.jpg" alt="Handball EM">
            <figcaption>Handball EM</figcaption>
        </figure>
        <p>TSG 1860 Hoffenheim im teamcheck: Eine Hinrunde mit Licht und Schatten</p>
    </div>
    <div class="col-3">
        <figure>
            <img src="images/handball3.jpg" alt="Handball EM">
            <figcaption>Handball EM</figcaption>
        </figure>
        <p>TSG 1860 Hoffenheim im teamcheck: Eine Hinrunde mit Licht und Schatten</p>
    </div>
</div>
</section>

<section class="main_content">
<header>
    <h2>Service</h2>
</header>
<div class="row">
    <div class="col-4">
        <figure>
            <img src="images/wetter.jpg" alt="Die Wettervorhersage">
            <figcaption>Aussichten</figcaption>
            <h4>Die Wettervorhersage</h4>
        </figure>
    </div>
    <div class="col-4">
        <figure>
            <img src="images/dax.jpg" alt="Aktuelle Börsenkurse">
            <figcaption>Marktüberblick</figcaption>
            <h4>Aktuelle Börsenkurse</h4>
        </figure>
    </div>
    <div class="col-4">
        <figure>
            <img src="images/tagesschau.jpg" alt="Tagesschau in 100 Sekunden">
            <figcaption>Nachrichten in Kürze</figcaption>
            <h4>Tagesschau in 100 Sekunden</h4>
        </figure>
    </div>
    <div class="col-4">
        <figure>
            <img src="images/audiothek.jpg" alt="ALs App im Web: die ARD">
            <figcaption>Doku, Hörspiel, Comedy - jederzeit das Beste hören.</figcaption>
            <h4>ALs App im Web: die ARD</h4>
        </figure>
    </div>
</div>
</section>

</main>
<footer>
    <div class="wrapper">
        <section>
            <div class="row">
                <nav class="col-4">
                    <h4>Service</h4>
                    <ul>
                        <li><a href="#">Zuschauerredaktion Das Erste</a></li>
                        <li><a href="#">ARD Mediathek APP</a></li>
                        <li><a href="#">ARD Audiothek APP</a></li>
                        <li><a href="">Sportschau APP</a></li>
                        <li><a href="#">Tagesschau APP</a></li>
                        <li><a href="#">DasErste APP</a></li>
                        <li><a href="#">Newsletter</a></li>
                        <li><a href="#">RSS Feed</a></li>
                        <li><a href="#">Twitter</a></li>
                        <li><a href="#">Facebook</a></li>
                        <li><a href="#">Google+</a></li>
                        <li><a href="#">YouTube</a></li>
                        <li><a href="#">Instagram</a></li>
                    </ul>
                </nav>
                <nav class="col-4">
                    <h4>ARD online</h4>
                    <ul>
                        <li><a href="#">ARD Startseite</a></li>
                        <li><a href="#">Nachrichten</a></li>
                        <li><a href="#">Sport</a></li>
                        <li><a href="#">Börse</a></li>
                        <li><a href="#">Ratgeber</a></li>
                        <li><a href="#">Wissen</a></li>
                        <li><a href="#">Kultur</a></li>
                        <li><a href="#">Kinder</a></li>
                        <li><a href="#">Die ARD</a></li>
                        <li><a href="#">Fernsehen</a></li>
                        <li><a href="#">Radio</a></li>
                        <li><a href="#">ARD Mediathek</a></li>
                    </ul>
                    
                </nav>
                <nav class="col-4">
                    <h4>Gemeinsame Programme</h4>
                    <ul>
                        <li><a href="#">Das Erste</a></li>
                        <li><a href="#">ONE</a></li>
                        <li><a href="#">Tagesschau24</a></li>
                        <li><a href="#">3sat</a></li>
                        <li><a href="#">arte</a></li>
                        <li><a href="#">funk</a></li>
                        <li><a href="#">KiKa</a></li>
                        <li><a href="#">phoenix</a></li>
                    </ul>
                </nav>
                <nav class="col-4">
                    <h4>Service</h4>
                    <ul>
                        <li><a href="#">Bayerischer Rundfunk</a></li>
                        <li><a href="#">Hessischer Rundfunk</a></li>
                        <li><a href="#">Mitteldeutscher Rundfunk</a></li>
                        <li><a href="#">Norddeutscher Rundfunk</a></li>
                        <li><a href="#">Radio Bremen</a></li>
                        <li><a href="#">Rundfunk Berlin-Brandenburg</a></li>
                        <li><a href="#">Saarländischer Rundfunk</a></li>
                        <li><a href="#">Südwestrundfunk</a></li>
                        <li><a href="#">Westdeutscher Rundfunk</a></li>
                        <li><a href="#">Deutsche Welle</a></li>
                        <li><a href="#">Deutschlandradio</a></li>
                    </ul>
                </nav>
            </div>
        </section>

        <section>
            <ul class="imprint">
                <li><a href="#">Impressum</a></li>
                <li><a href="#">Hilfe</a></li>
                <li><a href="#">Kontakt</a></li>
                <li><a href="#">Datenschutzbestimmungen</a></li>
                <li><a href="#">Nutzungsbedingungen</a></li>
                <li><a href="#">Bildnachweise</a></li>
            </ul>
        </section>
        <section class="copy">
            <p>
                &copy; <a href="#">MDR</a> | <a href="#">ARD.de</a>
            </p>
                <img src="images/ard_wse.png" alt="ARD">  
        </section>
    </div><!-- /.wrapper -->
</footer>
</body>
</html>